packages/fxa-settings/src/models/Account.ts

```diff
@@ -721,5 +722,6 @@ export class Account implements AccountData {
       )
     );
     firefox.accountDeleted(this.uid);
+    Storage.factory('localStorage').clear();
```

Comment:
Would this remove the currently signed in user or all users in local storage?

Reply:
It does remove all of them. IDK what the previous behavior was but it seems reasonable to me to clear the list at this point. If it isn't cleared the next visit defaults to the first account in the list, which seems surprising to me. I think its better to start from 'email first' after a 'delete account'. What do you think?

Reply:
> If it isn't cleared the next visit defaults to the first account in the list, which seems surprising to me.

Yea that is surprising. Would be nice to have an account chooser option but that is out of scope.

Let's keep this as is and show the user email first 👍🏽